server/lib/email.js

```diff
@@ -293,14 +293,8 @@ const generateEmailFromTemplate = (template, recipient, data = {}, options = {})
       template = 'thankyou.wwcode';
     } else if (['foundation', 'opensource'].includes(hostSlug)) {
       template = `thankyou.${hostSlug}`;
-    } else if (includes(['chsf', 'kendraio', 'brusselstogether', 'sustainoss', 'ispcwa'], slug)) {
```

Comment:
- `chsf` & `ispcwa` templates were already removed
- `brusselstogether` has [no financial activity](https://opencollective.com/brusselstogether/transactions) since its creation
- `sustainoss` template could not be triggered as it's now hosted by OSC, which triggered the other template above (`else if (['foundation', 'opensource'].includes(hostSlug))`)